module/solutions/helper.js

```diff
@@ -1603,6 +1603,16 @@ module.exports = class SolutionsHelper {
 
           }
 
+        } else if ( solutionData.type === constants.common.SURVEY ) {
+          // Get survey submissions of user
+          let surveySubmissionDetails = await surveyService.userSurveySubmissions(
```

Comment:
@ankitpws can you please add function signature, sample request/response please.

Reply:
Added